cypress/support/commands.ts

```diff
@@ -53,13 +76,18 @@ Cypress.Commands.add('loadPdf', (path: '/pdf' | '/pdf-collection') => {
     Cypress.config().baseUrl === 'http://localhost:1234'
       ? 'http://localhost:3001'
       : 'https://drb-api-qa.nypl.org/utils';
-  cy.intercept('GET', `${pdfProxyInterceptUrl}/**`).as('pdf');
+  cy.intercept(`${pdfProxyInterceptUrl}/**`, { middleware: true }, (req) => {
+    req.on('before:response', (res) => {
+      // force all API responses to not be cached
+      res.headers['cache-control'] = 'no-store';
+    });
+  }).as('pdf');
```

Comment:
this ensures we aren't caching resources in tests, which can cause cypress to not know that fetch requests have happened successfully